### PR TITLE
feat: change ordinals content url

### PR DESCRIPTION
**Pull request by alter-eggo**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/wallet/actions/runs/4456880165).

See title
Screenshot 2023-03-18 at 14 35 55

### CODE REVIEW

**Review by kyranjamie — DISMISSED** [review]
Code LGTM if QAd

**Review by markmhendrickson — DISMISSED** [review]

**Comment by fbwoolf** [comment]
I QA'd this build and I see my ordinals as expected...

Screen Shot 2023-03-18 at 9 28 31 AM

**Review by fbwoolf — DISMISSED** [review]

File: src/app/query/bitcoin/ordinals/utils.ts [hunk 1]
```diff
@@ -94,31 +94,50 @@ export function whenInscriptionType<T>(
...
+  content_type: string;
+  content_length: number;
+  timestamp: number;
+}
 
-  // Were HTML to page to change, thumbnailCount alone would dangerously return
-  // zero 0, hence additional check that inscrption header is also missing
-  if (thumbnailCount === 0 && firstHeaderText !== 'inscriptions') return 0;
+// Example:
+// https://ordapi.xyz/output/758bd2703dd9f0a2df31c2898aecf6caba05a906498c9bc076947f9fc4d8f081:0
```
Comment (fbwoolf) [inline src/app/query/bitcoin/ordinals/utils.ts]: This example needs to be updated?

File: src/app/features/collectibles/components/ordinals.tsx [hunk 2]
```diff
@@ -34,12 +34,12 @@ function Inscription({ path, utxo }: InscriptionProps) {
   const inscriptionMetadata = whenInscriptionType<InscriptionInfo>(inscription.content_type, {
     image: () => ({
       infoUrl: createInfoUrl(inscription.content),
-      src: `https://ordinals.com${inscription.content}`,
+      src: `https://ordapi.xyz${inscription.content}`,
```
Comment (fbwoolf) [inline src/app/features/collectibles/components/ordinals.tsx:37]: I think these are still outdated?

File: src/app/features/collectibles/components/ordinals.tsx [hunk 3]
```diff
@@ -34,12 +34,12 @@ function Inscription({ path, utxo }: InscriptionProps) {
...
       type: 'image',
       ...inscription,
     }),
     text: () => ({
-      contentSrc: `https://ordinals.com${inscription.content}`,
+      contentSrc: `https://ordapi.xyz${inscription.content}`,
```
Comment (fbwoolf) [inline src/app/features/collectibles/components/ordinals.tsx:42]: ?

File: src/app/query/bitcoin/ordinals/utils.ts [hunk 4]
```diff
@@ -94,31 +94,50 @@ export function whenInscriptionType<T>(
...
-
-  // First content on page is inscrption section header and thumbnail of
-  // inscrptions in utxo
-  const firstSectionHeader = utxoPage.querySelector('dl > dt:first-child');
-  if (!firstSectionHeader)
-    throw new Error('If no element matching this selector is found, something is wrong');
-
-  const firstHeaderText = firstSectionHeader.textContent;
-  const thumbnailCount = utxoPage.querySelectorAll('dl > dt:first-child + dd.thumbnails a').length;
+export interface OutputResult {
```
Comment (fbwoolf) [inline src/app/query/bitcoin/ordinals/utils.ts]: I think this interface name could be more descriptive?

**Review by edu-stx — APPROVED** [review]

**Pull request merged**